src/Command/Helpers/PlatformSitesTrait.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Acquia\Console\Cloud\Command\Helpers;
+
+use Acquia\Console\Acsf\Platform\ACSFPlatform;
+use Acquia\Console\Cloud\Platform\AcquiaCloudMultiSitePlatform;
+use Acquia\Console\Cloud\Platform\AcquiaCloudPlatform;
+use EclipseGc\CommonConsole\PlatformInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Trait PlatformSitesTrait.
+ *
+ * Used for fetching the platform sites.
+ *
+ * @package Acquia\Console\ContentHub\Command\Helpers
+ */
+trait PlatformSitesTrait {
+
+  /**
+   * Gets the uri of one of the sites.
+   *
+   * @param \EclipseGc\CommonConsole\PlatformInterface $platform
+   *   Platform object.
+   * @param \Symfony\Component\Console\Input\InputInterface $input
+   *   Input stream.
+   * @param \Symfony\Component\Console\Output\OutputInterface $output
+   *   Output stream.
+   *
+   * @return string
+   *   URI to return.
+   */
+  protected function getUri(PlatformInterface $platform, InputInterface $input, OutputInterface $output): string {
```

Comment:
We are mixing dependencies here. Why does PlatformSitesTrait reside in cloud-contenthub-console exactly? It acsf is not listed as dependency.

Reply:
This needs to be changed

Reply:
Moved it to Content Hub console helpers.